ddl/lightning/BUILD.bazel

```diff
@@ -0,0 +1,8 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+go_library(
```

Comment:
We don't need to enable flaky test because all the tests in this package are expected to be stable.

Reply:
> run gazelle again to update bazel config.

Updated.

Reply:
> We don't need to enable flaky test because all the tests in this package are expected to be stable.

we hope it is forever stable. But sometimes things don't go the way we think. we can enable it before happening.
BTW, enabling a flaky test will not affect the test speed.

Reply:
OK.